PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -154,9 +166,16 @@ struct StoreAppConditionalView: View {
                                     Rectangle()
                                          .fill(.regularMaterial)
                                          .overlay {
-                                             ProgressView()
-                                                 .progressViewStyle(.circular)
+                                             if loadingLocalIcon {
+                                                 ProgressView()
+                                                     .progressViewStyle(.circular)
+                                             } else {
+                                                 Image(systemName: "exclamationmark.triangle")
+                                                     .font(.system(size: 24))
```

Comment:
Setting the font isn't needed.

Reply:
> Setting the font isn't needed.

The purpose of setting a font is to specify a size for the icon, to ensure that the icon is of a reasonable size to the space available. Without specifying a size, the icon may appear too small to view, like so:


<img width="107" alt="截屏2024-06-26 上午11 35 13" src="https://github.com/PlayCover/PlayCover/assets/66894537/fe9350a5-0244-466f-8b57-91d323ca910f">


whereas with a font size specified:
<img width="94" alt="截屏2024-06-26 上午11 36 33" src="https://github.com/PlayCover/PlayCover/assets/66894537/6b523245-a891-4e11-b119-2042c10abfb8">


especially considering how the spinner icon is already of a relative large size, it would only make sense that they match.
<img width="100" alt="截屏2024-06-26 上午11 37 03" src="https://github.com/PlayCover/PlayCover/assets/66894537/72e1c8c6-1cea-4e13-a3ba-0404de43badd">

Let me know what you think!